modules/sno-du-configuring-the-operators.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-configuring-the-operators_{context}"]
+= Creating OperatorGroups for Operators
+
+This configuration is provided to enable addition of the Operators needed to configure the platform post installation. It adds the Namespaces and OperatorGroups for the Local Storage Operator, Logging Operator, Performance Addon Operator, PTP Operator, and the SRIOV Network Operator.
```

Comment:
Again, what is provided? Do we provide an example for the user to create the object? Or, is there a pre-configured object that the user needs to modify?

Reply:
changed syntax. @imiller0 Again, do we need further clarification here?

Reply:
These CRs are necessary to allow the user to subscribe to a set of Operators that are necessary for realizing the set of features and performance goals to run a DU application on the cluster. In order to install the operators a Namespace and OperatorGroup must be created for each. If it makes more sense you can group these with the Subscriptions that they are related to.